drud/ddev
# Enhance typo3 global command with options

## The Problem/Issue/Bug:

This patch adds the options -f|--first-install and -e|--enable-install-tool to create ${DOCROOT}/FIRST_INSTALL respectively ${DOCROOT}/typo3conf/ENABLE_INSTALL_TOOL which are often used to setup and configure a TYPO3 project.

The output of `ddev help typo3` is now:

```
Run TYPO3 CLI (typo3) command inside the web container (global shell web container command)

Usage:
  ddev typo3 [flags] [command] [options] [arguments]

Examples:
  ddev typo3 --first-install
  ddev typo3 list
  ddev typo3 site:list
  ddev typo3 extension:list

Flags:
  -e, --enable-install-tool   Enable the TYPO3 Install Tool by creating the file $DOCROOT/typo3conf/ENABLE_INSTALL_TOOL
  -f, --first-install         Enable the TYPO3 Install Wizard by creating the file $DOCROOT/FIRST_INSTALL
  -h, --help                  help for typo3

Global Flags:
  -j, --json-output   If true, user-oriented output will be in JSON format.
```

## Manual Testing Instructions:
- run `ddev typo3 -f` or `ddev typo3 --first-install` and check if ${DOCROOT}/FIRST_INSTALL is created
- run `ddev typo3 -e` or `ddev typo3 --enable-install-tool` and check if ${DOCROOT}/typo3conf/ENABLE_INSTALL_TOOL is created
